app/views/menus/show.html.erb

```diff
@@ -0,0 +1,71 @@
+<div class="edit-window">
+  <div class="section-container">
+    <% @menu.menu_sections.each do |s| %>
+      <li class="section-name section-<%=s.id%>">
+        <%= link_to s.name, show_section_path(:menu_id => @menu.id, :sec_id => s.id) %>
```

Comment:
Prefer the newer Ruby 1.9 hash style (no hash rockets), unless you have a reason not to:

```html
<%= link_to s.name, show_section_path(menu_id: @menu.id, sec_id: s.id) %>
```